src/prefs/MusicInformationRetrievalPrefs.cpp

```diff
@@ -0,0 +1,9 @@
+#include "MusicInformationRetrievalPrefs.h"
+
+ChoiceSetting UseMirResultToConfigureProject {
+   wxT("/GUI/UseMirResultToConfigureProject"),
+   { ByColumns,
+     { {}, {}, {} },
+     { wxString("Yes"), wxString("No"), wxString("Ask") } },
```

Comment:
Shouldn't be there a way to change that in preferences dialog?

Reply:
We haven't discussed that with Design. I would say so, only then I wouldn't know if checking "Don't ask me again" should modify the value in that other dialog. @LWinterberg ?